app/controllers/api/v1/client_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::V1::ClientController < ApplicationController
+
+  protect_from_forgery with: :null_session
+
+  def index
+    render json: Client.all, status: 200
```

Comment:
for demo purposes it's ok
But if you have many clients, this will load an entire table into memory, it may cause OutOfMemory exceptions